### PR TITLE
Update guillotina and aiohttp dani

### PR DESCRIPTION
**Pull request by dmanchon**
fixed the travis issues with https://github.com/guillotinaweb/guillotina_s3storage/pull/3

### CODE REVIEW

File: guillotina_s3storage/tests/mock_server.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
+import asyncio
```
Comment (bloodbare) [inline guillotina_s3storage/tests/mock_server.py]: Are we using this mock on travis ? Its not started before ?

File: guillotina_s3storage/tests/fixtures.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+class PatchedBaseRequest(aiohttp.web_request.Request):
+    @property
+    def content(self):
+        return self._payload
+
+
+@pytest.fixture(scope='function')
+def own_dummy_request(dummy_request):
```
Comment (bloodbare) [inline guillotina_s3storage/tests/fixtures.py:46]: Why we need it ?

**Review by bloodbare — CHANGES_REQUESTED** [review]

File: guillotina_s3storage/tests/mock_server.py (hunk 1 above)
Comment (dmanchon) [reply guillotina_s3storage/tests/mock_server.py]: yeah! you are totally right, we can delete it now

File: guillotina_s3storage/tests/fixtures.py (hunk 2 above)
Comment (dmanchon) [reply guillotina_s3storage/tests/fixtures.py:46]: The `PatchedBaseRequest` doesnt cache `_content` field so the request can be re-used.

**Review by lferran — APPROVED** [review]

**Review by bloodbare — APPROVED** [review]
@vangheem any opinion?

File: guillotina_s3storage/tests/mocks.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from unittest.mock import MagicMock
+
+
+class AsyncMock(MagicMock):
```
Comment (vangheem) [inline guillotina_s3storage/tests/mocks.py:4]: oh, nice way to make this work.

**Review by vangheem — APPROVED** [review]

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,8 +1,12 @@
-2.0.1 (unreleased)
+2.0.2 (unreleased)
 ------------------
 
 - Nothing changed yet.
 
+2.0.1 (2018-17-09)
```
Comment (vangheem) [inline CHANGELOG.rst]: You don't make the release here!

This is something done outside the PR cycle.

**Review by vangheem — CHANGES_REQUESTED** [review]
Can you please fix the changelog?

**Comment by vangheem** [comment]
Looks great! Thanks for making this work!

Can you please just fix the changelog? I'll merge then.

**Review by vangheem — APPROVED** [review]

**Comment by vangheem** [comment]
Updated and merged. 2.0.1 release made.

**Pull request merged**